media/js/foundation/annual2011.js

```diff
@@ -215,29 +210,21 @@
 
     // Set up the contributor stories carousel
     function setupCarousel() {
-        $sliderPrime.clone().attr("id", "story-slider-clone").insertAfter($("#video-stage"));
+        $('.jcarousel-clip').jcarousel();
         if ( $("#story-vid").length > 0 ) {
             $("#story-vid")[0].pause();
         }
-        $sliderPrime.hide();
-        $("#story-slider-clone").jcarousel({
-            scroll: 4,
-            visible: 4,
-            buttonNextHTML: null,
-            buttonPrevHTML: null,
-            itemLastOutCallback: { onAfterAnimation: disableButtons },
-            itemLastInCallback: { onAfterAnimation: disableButtons },
-            initCallback: controlButtons
-        });
+        $sliderPrime.removeAttr("style");
```

Comment:
Hm, I don't think you want to make this call here - it seems it should be called when moving into a mobile sized viewport. Probably in the `matchMedia` listener above?